flocker/ca/_script.py

```diff
@@ -16,8 +16,76 @@
 from ..common.script import (flocker_standard_options, ICommandLineScript,
                              FlockerScriptRunner)
 
-from ._ca import (CertificateAuthority, CertificateAlreadyExistsError,
-                  KeyAlreadyExistsError)
+from ._ca import (CertificateAuthority, ControlCertificate,
+                  CertificateAlreadyExistsError, KeyAlreadyExistsError,
+                  PathError)
+
+
+class ControlCertificateOptions(Options):
+    """
+    Command line options for ``flocker-ca create-control-certificate``.
+    """
+
+    longdesc = """Create a new certificate for the control service.
+
+    Creates a certificate signed by a previously generated certificate
+    authority (see flocker-ca initialize command for more information).
+
+    The certificate will be stored in the current working directory.
+
+    Parameters:
+
+    * authoritypath: Path to the root certificate's private key. Defaults to
+    ./cluster.key
+    * certfile: path to the root certificate file. Defaults to ./cluster.crt
+    """
+
+    synopsis = "<authoritypath>"
+
+    def parseArgs(self, *args):
+        self["path"] = FilePath(os.getcwd())
+        self["rootpath"] = FilePath(os.getcwd())
+        if len(args) == 1:
+            self["rootpath"] = FilePath(args[0])
```

Comment:
This doesn't seem to match the synopsis or the documentation above, which talk about authoritypath and certfile.

And if those are really optional, they should be defined as command line options rather than positional arguments.